docs/developer/nextflow.k8s.md

```diff
@@ -14,4 +14,4 @@ Some classes may be excluded from the above diagram for brevity.
 
 ## Notes
 
-The Kubernetes integration uses the K8s REST API to interact with K8s clusters, and relies on the `kubectl` command and `~/.kube/config` file for authentication.
+The Kubernetes integration uses the K8s HTTP API to interact with K8s clusters, and relies on the `kubectl` command and `~/.kube/config` file for authentication.
```

Comment:
`REST API` is fine https://kubernetes.io/docs/reference/using-api/

Reply:
REST is overused, most APIs that claim to be REST APIs are not actually REST

Reply:
See this post from the creator of REST: https://roy.gbiv.com/untangled/2008/rest-apis-must-be-hypertext-driven

It is a minor point but I'm trying to not perpetuate the error

Reply:
Fair enough